routes/index.js

```diff
@@ -6,9 +6,7 @@ var fs = require('fs');
 var Cart = require('../models/cart');
 var products = JSON.parse(fs.readFileSync('./data/products.json', 'utf8'));
 
-router.get('/', function (req, res, next) {
-  var productId = products && products[0].id;
-
+router.get('/', function (_req, res, _next) {
```

Comment:
good work!

Reply:
**bold** __italics__

Reply:
```
foo
```

Reply:
abc

Reply:
ctrl + enter to submit

Reply:
```css
.foo {
    background: white;
}
```

code block

Reply:
```css
.bar {
	background: black;
}
```

Reply:
<script>alert(123)</script>

Reply:
&lt;script&gt;alert(&#39;123&#39;);&lt;/script&gt;

Reply:
<img src=x onerror=alert(123) />

Reply:
<svg><script>123<1>alert(123)</script>

Reply:
"><script>alert(123)</script>

Reply:
'><script>alert(123)</script>

Reply:
><script>alert(123)</script>

Reply:
</script><script>alert(123)</script>

Reply:
< / script >< script >alert(123)< / script >

Reply:
onfocus=JaVaSCript:alert(123) autofocus

Reply:
" onfocus=JaVaSCript:alert(123) autofocus

Reply:
' onfocus=JaVaSCript:alert(123) autofocus

Reply:
＜script＞alert(123)＜/script＞

Reply:
<sc<script>ript>alert(123)</sc</script>ript>

Reply:
--><script>alert(123)</script>

Reply:
";alert(123);t="

Reply:
';alert(123);t='

Reply:
JavaSCript:alert(123)

Reply:
;alert(123);

Reply:
src=JaVaSCript:prompt(132)

Reply:
"><script>alert(123);</script x="

Reply:
'><script>alert(123);</script x='

Reply:
><script>alert(123);</script x=

Reply:
" autofocus onkeyup="javascript:alert(123)

Reply:
' autofocus onkeyup='javascript:alert(123)

Reply:
<a href="\x0Bjavascript:javascript:alert(1)" id="fuzzelement1">test</a>

Reply:
**bold**

Reply:
<img \x00src=x onerror="alert(1)">

Reply:
<img \x47src=x onerror="javascript:alert(1)">

Reply:
![alt text](https://github.com/adam-p/markdown-here/raw/master/src/common/images/icon48.png "Logo Title Text 1")

Reply:
| Tables        | Are           | Cool  |
| ------------- |:-------------:| -----:|
| col 3 is      | right-aligned | $1600 |
| col 2 is      | centered      |   $12 |
| zebra stripes | are neat      |    $1 |

Reply:
<dl>
  <dt>Definition list</dt>
  <dd>Is something people use sometimes.</dd>

  <dt>Markdown in HTML</dt>
  <dd>Does *not* work **very** well. Use HTML <em>tags</em>.</dd>
</dl>

Reply:
<a href="http://www.youtube.com/watch?feature=player_embedded&v=YOUTUBE_VIDEO_ID_HERE
" target="_blank"><img src="http://img.youtube.com/vi/YOUTUBE_VIDEO_ID_HERE/0.jpg" 
alt="IMAGE ALT TEXT HERE" width="240" height="180" border="10" /></a>

Reply:
[![IMAGE ALT TEXT HERE](http://img.youtube.com/vi/YOUTUBE_VIDEO_ID_HERE/0.jpg)](http://www.youtube.com/watch?v=YOUTUBE_VIDEO_ID_HERE)

Reply:
[![IMAGE ALT TEXT HERE](http://img.youtube.com/vi/YOUTUBE_VIDEO_ID_HERE/0.jpg)](javascript:alert(1))

Reply:
[![IMAGE ALT TEXT HERE](http://img.youtube.com/vi/YOUTUBE_VIDEO_ID_HERE/0.jpg)](javascript:alert(1))

Reply:
[![IMAGE ALT TEXT HERE](http://img.youtube.com/vi/YOUTUBE_VIDEO_ID_HERE/0.jpg)](http://www.youtube.com/watch?v=YOUTUBE_VIDEO_ID_HERE)